infrastructure/variables.tf

```diff
@@ -146,6 +146,10 @@ variable "elastic_search_enabled" {
   default = "false"
 }
 
+variable "elastic_search_timeout" {
+  default = "6000"
```

Comment:
let's set to 10 sec please, and let's remove the explicit setting on prod.tfvars

Reply:
DONE